meal-planner/src/components/MealSelectMenu.js

```diff
@@ -1,24 +1,43 @@
 import styled from 'styled-components/macro'
-import { v4 as uuidv4 } from 'uuid'
-
-export default function MealSelectMenu({ weekdayId, handleMealClick }) {
-    const meals = [
-        { meal: 'Käsestulle', id: uuidv4() },
-        { meal: 'Kartoffelsalat', id: uuidv4() },
-        { meal: 'Gemüselasagne', id: uuidv4() },
-        { meal: 'Steinpilzrisotto', id: uuidv4() },
-        { meal: 'Currywurst', id: uuidv4() },
-        { meal: 'Linsensuppe', id: uuidv4() },
-        { meal: 'Tofufrikassee', id: uuidv4() },
-        { meal: 'Banana Pancakes', id: uuidv4() },
-        { meal: 'Thai Gemüsecurry', id: uuidv4() },
-    ]
+import meals from '../data/recipeData.json'
 
+export default function MealSelectMenu({
+    weekdayId,
+    handleMealClick,
+    handleRecipeClick,
+}) {
     return (
         <ListStyled>
-            {meals.map(({ meal, id }) => (
-                <li key={id} onClick={() => handleMealClick(meal, weekdayId)}>
+            {meals.map(({ meal, id, ingredients, instructions, image }) => (
+                <li key={id}>
                     {meal}
+                    <div>
+                        <button
+                            onClick={() =>
+                                handleMealClick(
+                                    meal,
+                                    weekdayId,
+                                    ingredients,
+                                    instructions,
+                                    image
+                                )
+                            }
+                        >
+                            ok
```

Comment:
Maybe something like "Auswählen" or "Übernehmen" would be more clear what it does and sound and look more professional.

Reply:
good point